File: 合并两个有序数组/C++/original.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        for(int i=m; i<m+n; ++i){
            nums1[i]=nums2[i-m];
        }
        sort(nums1.begin(), nums1.begin()+(m+n));
    }
};
/* 思路：通过for循环把nums的有效元素赋给nums1的有效元素后面，
 * 之后用sort函数对nums1进行排序。
 */

File: 题库/4. 寻找两个有序数组的中位数/python/original.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        '''
        思路：把两个数组拼接起来，之后把数组排序，
        判断数组的数量奇偶，如果为奇数，返回中间的
        数就可以，如果为偶数，返回中间两个数的平均数。
        '''
        nums = nums1 + nums2
        nums.sort()
        n = len(nums)
        if n%2 == 0:
            return (nums[n//2]+nums[n//2-1])/2
        else:
            return nums[n//2]